CMakeLists.txt

```diff
@@ -169,6 +169,7 @@ list(APPEND EXTERNAL_LIBS Threads::Threads)
 list(APPEND EXTERNAL_LIBS ${Backtrace_LIBRARY})
 list(APPEND EXTERNAL_LIBS xxhash)
 list(APPEND EXTERNAL_LIBS span-lite)
+list(APPEND EXTERNAL_LIBS stdc++fs)
```

Comment:
Is this test only, or we may using it outside testing?

Reply:
done